tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -399,6 +399,60 @@ def test_choice_invalid_value(self):
             self.rs.choice(a=self.a, size=self.size, p=self.p)
 
 
+@testing.parameterize(
+    {'a': 5, 'size': 2},
+    {'a': 5, 'size': (2, 2)},
+    {'a': 5, 'size': ()},
+    {'a': numpy.array([0.0, 2.0, 4.0]), 'size': 2},
+)
+@testing.fix_random()
+@testing.with_requires('numpy>=1.11.0')
+@testing.gpu
+class TestChoiceReplaceFalse(unittest.TestCase):
+
+    def setUp(self):
+        self.rs = cupy.random.get_random_state()
+        self.rs.seed(testing.generate_seed())
+
+    def test_dtype_shape(self):
+        v = self.rs.choice(a=self.a, size=self.size, replace=False)
+        if isinstance(self.size, six.integer_types):
+            expected_shape = (self.size,)
+        else:
+            expected_shape = self.size
+        if isinstance(self.a, numpy.ndarray):
+            expected_dtype = 'float'
+        else:
+            expected_dtype = 'int64'
```

Comment:
This test has no error using 'int' instead of 'int64' on both of ubuntu and Windows.

Reply:
ok